### DISCUSSION TITLE
Is there a way to center all widgets in the qtile bar?

### DISCUSSION THREAD

**Discussion by pranavsetpal** [qtile/qtile#3996] [Q&A]
EDIT: Is there a way to **vertically align** all widgets in the qtile bar?
The question is pretty self-explanatory, could I center all widgets in the bar?
I know it might be possible to make it centered via the padding/margin though I wasn't able to find a way to do it. More so, it seems very hacky and would simply break down if I change the bar height.

**Comment by elParaguayo** [comment]
Put a `Spacer` widget at the start and end your bar.

**Reply by pranavsetpal** [reply]
I meant vertically align them, not center sorry. Added an edit in body for the same

**Reply by elParaguayo** [reply]
Which one's aren't vertically centred? Can you post a screenshot?

**Reply by pranavsetpal** [reply]
I am an idiot, honestly, there are some extremely small bezels on my screen that cause it to look not aligned.

I really need to fact check before posting anything. Really sorry

**Reply by elParaguayo** [reply]
Not a problem at all.
